frame/support/Cargo.toml

```diff
@@ -56,3 +56,4 @@ std = [
 nightly = []
 strict = []
 runtime-benchmarks = []
+assert-require-transaction = []
```

Comment:
Please remove. You can enable `debug_assertions` in release builds. (Substrate is doing this for example in CI)